Classes/Emogrifier.php

```diff
@@ -430,6 +431,26 @@ protected function process(\DOMDocument $xmlDocument)
     }
 
     /**
+     * This method searches for all nodes with a style attribute, transforms the CSS found
+     * to HTML attributes and adds those attributes to each node
```

Comment:
Just to make this perfect, could you please add the period at the end of the sentence and remove "This method" from the description (so that it starts with "Searches for …"?